### DISCUSSION TITLE
open bullets for nested lsts

### DISCUSSION THREAD

**Discussion by titaniumbones** [minad/org-modern#167] [Q&A]
How hard would it be for me to add the ability to use a different symbol for *nested* list item bullets? I'm used to this visual cue in other places I use at work (Slack, etc) and I find I miss it now that I'm using org-modern all the time in org.  thanks!

**Comment by minad** [comment]
The difficulty is that we don't know the level of the list item when fontifying the line. Such context sensitivity would require fontifying the whole list at once.

**Comment by titaniumbones** [comment]
Don’t we know the level from the number of spaces? I always thought that
was the determinant.

**Reply by minad** [reply]
You are right that we could use that if the indentation depth is always consistent.

~~~
- a
  - a
    - a
    - a
- b
    - b
        - b
        - b 
~~~

Why not use different symbols for lists in the text, e.g., alternating - and +? This should produce the desired effect.

~~~
- a
  + a
    - a
      + a
~~~

**Reply by titaniumbones** [reply]
huh osmehow I missed this.  I'm oten moving things around from one level to another using keystrokes, so while your solution absolutely works, it doesn't seem wright for my workflow. I made some modifications to your code at one point that was fixing hte issue for me, but I seem to have  allowed myself to overwrite it . 

Anyway, my solution was to hardcode some values into `org-modern--make-font-lock-keywords`.  It wouldn't work for the package, but is great for me.
